cypress/index.d.ts

```diff
@@ -1,6 +1,7 @@
 declare namespace Cypress {
   interface Chainable {
     waitForStableDOM(iteration?: number): void;
+    findExplorerNode(name: string): ReturnType<typeof cy.findByRole>;
```

Comment:
New command to find an explorer node without selecting it. I used it to check if a node was indeed selected in the explorer after selecting it in the search panel.